tests/unit/components/accordion-with-image/AccordionWithImage.spec.jsx

```diff
@@ -0,0 +1,29 @@
+import AccordionWithImage from '~/components/accordion-with-image/AccordionWithImage.jsx'
+import { screen, render, fireEvent } from '@testing-library/react'
+import { expect } from 'vitest'
```

Comment:
Import from libraries first. 
```suggestion
import { expect } from 'vitest'
import { screen, render, fireEvent } from '@testing-library/react'
import AccordionWithImage from '~/components/accordion-with-image/AccordionWithImage.jsx'
```